src/docs/pages/introduction/getting-started/react.md

```diff
@@ -0,0 +1,179 @@
+---
+eleventyNavigation:
+  key: React
+  parent: Getting started
+  order: 1
+layout: layout.njk
+title: React
+permalink: introduction/getting-started/react/
+---
+
+# React
+
+Liquid Oxygen provides React Bindings for all web components. Although the rendered components are still web components, the bindings improve the developer experience by providing a more familiar API and a better integration with React.
+
+On this page, you'll find detailed instructions on how to integrate Liquid Oxygen into your React project and how to use the components.
+
+## Prerequisites
+
+Liquid Oxygen is easy to integrate into an existing React project. We assume you already have a React project set up. If not, you can create a new project with vite.
+
+```sh
+npm create vite@latest your-project-name -- --template react-ts
+```
+
+For more information about this command and vite, please refer to the [official vite documentation](https://vitejs.dev/guide/#scaffolding-your-first-vite-project).<br />Although this guide assumes your project is using typescript, all examples should also be applicable to javascript projects.
+
+## Install
+
+Add Liquid Oxygen to your project with the package manager of your choice.
+
+```sh
+npm install @emdgroup-liquid/liquid
+```
+
+> Liquid Oxygen works with any package manager (e.g. npm, yarn or pnpm). For simplicity, we continue to use npm for the following guide.
+
+### Import and register web components
+
+The web components need to be registered before they can be used. Liquid Oxygen provides the `defineCustomElements` function to register all Liquid Oxygen components. The script loads components lazily only when used in your application. This reduces your bundle size and only loads required code at runtime.
```

Comment:
This is wrong. The cause is confusing terminology from Stencil's side. The components are defined lazily, not loaded lazily. The `defineCustomElements` method is only a convenience method for not defining each component individually. However, it does not have any effect on the bundle size. Nevertheless, the bundler can still tree shake the code, independently of whether the `defineCustomElements` method has been used or not.

```suggestion
The web components need to be registered before they can be used. Liquid Oxygen provides the `defineCustomElements` function to register all available Web Components. The script defines the components lazily, only when used in your application. 
```

Reply:
Think we talked about that last week after your comment:
So based on my observation the mechanics are something like this:

1. default dist: the consuming client registers all components with `defineCustomElements`. Once the browser finds the element in the dom it loads the corresponding script.

2. dist-custom-elements: requires the consuming client to import and register elements individually. This happens when importing the component (due to auto register). This shifts loading the required script for the component to the moment the the component got imported and does not wait (lazily) until it's actually rendered in the dom. 

I think we can omit the details and focus on "recommendations".